cilium/cilium.io
# refactor: pagination update

This pull request standardized the pagination by implementing a single, reusable component across various pages. This change aims to enhance code maintainability and ensure a consistent user experience.

Steps to check:

- open preview [blog](https://deploy-preview-395--cilium.netlify.app/blog/) page, [events](https://deploy-preview-395--cilium.netlify.app/events/) page, [labs](https://deploy-preview-395--cilium.netlify.app/labs/) page
- make sure that everything looks and feels as expected
